app/views/errors/StandardError.scala.html

```diff
@@ -14,22 +14,15 @@
  * limitations under the License.
  *@
 
-@this(govUkWrapper: GovUkWrapper)
+@this(mainTemplate: MainTemplate)
 
-@(appConfig: config.AppConfig, pageTitle: String, heading: String, message: String)(implicit request: Request[_], messages: Messages, user: Option[User] = None)
+@(appConfig: config.AppConfig,
+  pageTitle: String,
+  heading: String,
+  message: String)(implicit request: Request[_], messages: Messages)
 
-@contentHeader = {
- <h1>@messages(heading)</h1>
-}
+@mainTemplate(title = messages(pageTitle), appConfig = appConfig) {
+  <h1 class="govuk-heading-l">@messages(heading)</h1>
```

Comment:
Didn't catch this last time but can you just change this to be "xl" instead of "l"